wdt.c

```diff
@@ -37,8 +37,6 @@ int fd = -1;
 
 int pv_wdt_start(struct pantavisor *pv)
 {
-	int timeout = pv_config_get_watchdog_timeout();
-	int interval = pv_config_get_updater_interval();
```

Comment:
I have some doubts with this variable, if you see in the original code, the value from `pv_config_get_updater_interval()` was never used, but it could be, right? the solution only mimics the previous behavior, but is (and was) a bug, or not?

Reply:
I guess we put that on mistake in there after a config refactor. Previous to that, the config updater.interval was overridden based on the timeout value, which is not documented and a hidden weird behavior. I'd remove the "int interval = timeout / 2;" line, which is not doing anything, and the "pv_log(INFO..." that follows

Reply:
So, we remove all traces of `interval` in that function?

Reply:
ok